### PR TITLE
Add VIA keymap for Kawii9

### PR DESCRIPTION
**Pull request by kiwikey**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

**Comment by kiwikey** [comment]
Hi @fauxpark, there is an error compiling this VIA keymap, due to **BLUETOOTH_ENABLE = yes** in [rules.mk](https://github.com/qmk/qmk_firmware/blob/master/keyboards/kiwikey/kawii9/rules.mk). As I know, this is because bluetooth shifts the keycodes, which makes keycodes in VIA not to line up.

There are 2 solutions:
1) Change to **BLUETOOTH_ENABLE = no"**
2) Add **#define VIA_HAS_BROKEN_KEYCODES** to config.h

In my opinion, the 1st solution is better (cleaner code). Which solution do you prefer?

P.s: the reason of "BLUETOOTH_ENABLE = yes" is that the keyboard can work without plugging in to computer, to use as a bed lamp. The Kawii9 itseft has no bluetooth.

**Comment by fauxpark** [comment]
The second solution is for exactly this situation. However, if the board has no Bluetooth, then it can be disabled - what is probably allowing it to run without a USB connection is `NO_USB_STARTUP_CHECK` [(it is implicitly added when Bluetooth is enabled)](https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common.mk#L112), which you can set to `yes` in rules.mk.

**Comment by kiwikey** [comment]
Cool, then I will disable bluetooth and add `NO_USB_STARTUP_CHECK = yes` to rules.mk, I will open an PR for this.

Thank you very much for helping!

**Comment by fauxpark** [comment]
You can do that in this PR, which will allow the build to succeed.

File: keyboards/kiwikey/kawii9/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+BLUETOOTH_ENABLE = no
+NO_USB_STARTUP_CHECK = yes
```
Comment (fauxpark) [inline keyboards/kiwikey/kawii9/keymaps/via/rules.mk:3]:
```suggestion
```
Should go in the top level rules.mk

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by kiwikey** [comment]
The top level `rules.mk` was editted.
The `keymaps/via/rules.mk` was roll back to original (with only 1 line to enable VIA).

It seems ok now (I hope so).

**Review by fauxpark — APPROVED** [review]

**Review by Erovia — APPROVED** [review]

**Pull request merged**